classes/CategoryHelper.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Contao Open Source CMS
+ *
+ * Copyright (c) 2017 Heimrich & Hannot GmbH
+ *
+ * @author  Rico Kaltofen <r.kaltofen@heimrich-hannot.de>
+ * @license http://www.gnu.org/licences/lgpl-3.0.html LGPL
+ */
+
+namespace NewsCategories;
+
+class CategoryHelper
+{
+
+    /**
+     * Prepare the category object
+     *
+     * @param NewsCategoryModel $objCategory
+     *
+     * @return object
+     */
+    public static function prepareCategory(NewsCategoryModel $objCategory)
+    {
+        $category                = (object) $objCategory->row();
```

Comment:
I think the array gives much more possibilities as you can use all the related functions such as `foreach`, `array_map`, etc. Can you change it to array please?